src/System.Net.NameResolution/src/System.Net.NameResolution.csproj

```diff
@@ -10,6 +10,9 @@
     <OutputType>Library</OutputType>
```

Comment:
Since `Library` is default now, are you going to remove this `<outputType>` element, similar to the other netcore50 cleanup changes?